packages/iframe-coordinator/src/client.ts

```diff
@@ -355,34 +363,25 @@ bad input into one of the iframe-coordinator client methods.
 
     this._clientWindow.addEventListener('message', this._onWindowMessage);
     this._clientWindow.addEventListener('keydown', this._onKeyDown);
+    this._clientWindow.addEventListener('click', this._onWindowClick);
     this._sendToHost(Lifecycle.startedMessage);
   }
 
   /**
-   * Adds a global click handler to the client window that intercepts clicks on anchor elements
+   * Allows the click handler on the client window to intercept clicks on anchor elements
    * and makes a nav request to the host based on the element's href. This should be
    * avoided for complex applications as it can interfere with things like download
    * links that you may not want to intercept.
    */
   public startInterceptingLinks(): void {
-    if (this._isInterceptingLinksStarted) {
-      return;
-    }
-
-    this._isInterceptingLinksStarted = true;
-    this._clientWindow.addEventListener('click', this._onWindowClick);
+    this._shouldInterceptLinks = true;
   }
 
   /**
-   * Removes the global click handler that intercepts clicks on anchor elements.
+   * Turns off the behavior of intercepting link clicks in the client window click handler.
    */
   public stopInterceptingLinks(): void {
-    if (!this._isInterceptingLinksStarted) {
-      return;
-    }
-
-    this._isInterceptingLinksStarted = false;
-    this._clientWindow.removeEventListener('click', this._onWindowClick);
+    this._shouldInterceptLinks = true;
```

Comment:
Should this be set to false? Otherwise, both this and the startInterceptingLinks function do the same thing, which makes me wonder if there is a point to them at all

Reply:
Good catch!